attributes/sdk.rb

```diff
@@ -138,9 +138,15 @@
     "#{node['cdap']['sdk']['install_path']}/#{node['cdap']['sdk']['product_name']}/bin/cdap sandbox"
   end
 default['cdap']['sdk']['init_actions'] = [:enable, :start]
-# Get proper Node.js version on SDK 4.0+
-if node['cdap']['version'].to_f >= 4.0
+# Get proper Node.js version on SDK 4.x
+if node['cdap']['version'].to_f >= 4.0 && node['cdap']['version'].to_f < 5.0
```

Comment:
We should stop using `.to_f` for version comparisons everywhere as it's incorrect (4.10 < 4.2).  In this case we can just use `.to_i >= 4`, `.to_i < 5`